tests/baselines/reference/findAllRefsObjectBindingElementPropertyName06.baseline.jsonc

```diff
@@ -13,7 +13,7 @@
 // for (<|var { [|property1|]: p1 } of elems|>) {
 // }
 // var p2;
-// for (<|{ [|{| isWriteAccess: true |}property1|] : p2 } of elems|>) {
+// for (<|{ [|property1|] : p2 } of elems|>) {
```

Comment:
Could anyone please explain the meaning of "isWriteAccess" in this context? Additionally, I'm curious to know if this qualifies as a regression.

Reply:
in find-all-references, we can indicate that certain references are writes so that editors can filter reads and writes out, or display them differently. Technically this is correct as `property1` s not being written to - `p2` is.

Reply:
Though I wonder if `p2` has `isWriteAccess` on it.